views/index.hbs

```diff
@@ -1,19 +1,16 @@
 <div class="container-fluid">
 	<div class="grid">
       <div class="grid-sizer col-md-6"></div>
-			<div class="grid-item col-md-12">
-        <div class="grid-item-content" id="component-acolytes">
+			<div class="grid-item col-md-12" id="component-acolytes">
```

Comment:
formatting still seems wonky, but it was before, too, so can't really fault you on that.

Reply:
I just joined 2 divs here since inner one had no formatting. And with 2 divs Packery couldn't manage hidden components well (only acolyte one because it's wider but still). I left tabs/spaces alone just to make it clear what's changed and what's not.